package.json

```diff
@@ -57,6 +57,7 @@
     "react-dom": "18.0.0",
     "react-native": "0.69.5",
     "react-native-countdown-circle-timer": "^3.0.9",
+    "react-native-crypto-js": "^1.0.0",
```

Comment:
As we are already using `aes-js` in the code (see `src/lib/secure-storage/aes.ts`) maybe we could avoid adding another dependency.

Reply:
After some investigation and testing, with the current implementation of the aes-js we would need to install another package anyway (scrypt or bcrypt) to convert a `string` to a valid `key` for aes-js encryption function, so we would end up exactly the same, the current implementation it's pretty straight forward and the package is made for  RN. Also, the package has the AES algorithm as well if we need it. (This is also based on a discussion with @simoneb)